lib/internal/streams/add-abort-signal.js

```diff
@@ -23,11 +22,14 @@ function isStream(obj) {
   return !!(obj && typeof obj.pipe === 'function');
 }
 
-module.exports = function addAbortSignal(signal, stream) {
+module.exports.addAbortSignal = function addAbortSignal(signal, stream) {
   validateAbortSignal(signal, 'signal');
   if (!isStream(stream)) {
     throw new ERR_INVALID_ARG_TYPE('stream', 'stream.Stream', stream);
   }
+  return module.exports.addAbortSignalNoValidate(signal, stream);
+};
+module.exports.addAbortSignalNoValidate = function(signal, stream) {
```

Comment:
Shouldn't this return if signal is not valid?

Reply:
Why?

Reply:
Because now it will crash

Reply:
I didn't understand you the first time, you're right - that's a bug. Fixing.